File: Java/OOP/Inheritance/Single inehritance (example 1).java
class Teacher {
    String teacher_name = "James Gosling";
    String subject = "Java Programming";
}

class Student extends Teacher {
    String student_name = "Abdullah Al Naiem";
    int id = 123456;

    void displayInfo() {
        System.out.println("Teacher Name: " + teacher_name);
        System.out.println("Student Name: " + student_name);
        System.out.println("Student ID  : " + id);
        System.out.println("Class of    : " + subject);
    }
}

class singleInhertance {
    public static void main(String[] args) {
        Student obj = new Student();
        obj.displayInfo();
    }
}

File: Python/OOP/Inheritance/Single inehritance (example 1).py
class Teacher:
    teacher_name = "Guido van Rossum"
    subject = "Python Programming"


class Student(Teacher):
    student_name = "Abdullah Al Naiem"
    id = 123456

    def displayInfo(self):
        print("Teacher Name:", self.teacher_name)
        print("Student Name:", self.student_name)
        print("Student ID  :", self.id)
        print("Class of    :", self.subject)


class singleInheritance:
    obj = Student()
    obj.displayInfo();